src/qt/transactiontablemodel.cpp

```diff
@@ -391,24 +392,52 @@ QString TransactionTableModel::formatTxType(const TransactionRecord *wtx) const
     case TransactionRecord::Generated:
         return tr("Mined");
     case TransactionRecord::NameOp:
+    {
+        QString namespaceStrCap;
+        QString namespaceStrLow;
+        switch(wtx->nameNamespace)
+        {
+        case NameNamespace::Domain:
+            namespaceStrCap = tr("Domain");
+            namespaceStrLow = tr("domain");
```

Comment:
Do we really need both versions here?  I think it would be clearer to use `toLower` and only one hard-coded string, unless there is a particular Qt reason why this isn't possible.

Reply:
My understanding is that the localization scripts use a static analyzer to scan the source code for translatable strings; generating the English strings at run-time would prevent that static analyzer from finding the strings.  Since not all natural languages use case the same way as English, we also can't apply `toLower` to the post-localization strings.